static/app/views/projectInstall/messagingIntegrationAlertRule.tsx

```diff
@@ -0,0 +1,122 @@
+import {useEffect, useMemo} from 'react';
+import styled from '@emotion/styled';
+
+import SelectControl from 'sentry/components/forms/controls/selectControl';
+import Input from 'sentry/components/input';
+// import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {OrganizationIntegration} from 'sentry/types/integrations';
+import {
+  type IssueAlertNotificationProps,
+  providerDetails,
+} from 'sentry/views/projectInstall/issueAlertNotificationOptions';
+
+type Props = {
+  notificationProps: IssueAlertNotificationProps;
+  providersToIntegrations: Record<string, OrganizationIntegration[]>;
+};
+
+export default function MessagingIntegrationAlertRule({
+  notificationProps,
+  providersToIntegrations,
+}: Props) {
+  const {channel, integration, provider, setChannel, setIntegration, setProvider} =
+    notificationProps;
+
+  const providerOptions = useMemo(
+    () =>
+      Object.keys(providersToIntegrations).map(p => ({
+        value: p,
+        label: providerDetails[p].name,
+      })),
+    [providersToIntegrations]
+  );
+  const integrationOptions = useMemo(
+    () =>
+      provider && providersToIntegrations[provider]
+        ? providersToIntegrations[provider]?.map(i => ({
+            value: i,
+            label: i.name,
+          }))
+        : [],
+    [providersToIntegrations, provider]
+  );
+
+  useEffect(() => {
+    const providerKeys = Object.keys(providersToIntegrations);
+    if (providerKeys.length > 0) {
+      const firstProvider = providerKeys[0];
+      setProvider(firstProvider);
+
+      const firstIntegration = providersToIntegrations[firstProvider][0];
+      setIntegration(firstIntegration);
+    }
+  }, [providersToIntegrations, setProvider, setIntegration]);
+
+  if (!provider) {
+    return null;
+  }
+
+  return (
+    <RuleWrapper>
+      <Rule>
+        {providerDetails[provider]?.makeSentence({
+          providerName: (
+            <InlineSelectControl
+              aria-label="provider"
```

Comment:
```suggestion
              aria-label={t("Provider")}
```

Remember screen readers read these so they should also be translated!